plugins/polio/js/src/domains/Calendar/campaignCalendar/types.ts

```diff
@@ -1,14 +1,10 @@
+/* eslint-disable camelcase */
 import { Moment } from 'moment';
-import {
-    Campaign,
-    MergedShape,
-    Round,
-    Scope,
-    Shape,
-} from '../../../constants/types';
+import { Campaign, MergedShape, Scope, Shape } from '../../../constants/types';
+import { CampaignType } from '../../Campaigns/hooks/api/useGetCampaigns';
 
 export type Query = {
-    queryKey: any[];
+    queryKey: (string | Record<string, string>)[];
```

Comment:
```suggestion
    queryKey: (string | Record<string, unknown>)[];
```

You can pass add objects in the queryKey array